libs/sdk-ui-dashboard/src/presentation/dashboard/components/OverlayProvider.tsx

```diff
@@ -0,0 +1,147 @@
+// (C) 2022 GoodData Corporation
+import React, { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { OverlayController, OverlayControllerProvider } from "@gooddata/sdk-ui-kit";
+import { v4 as uuid } from "uuid";
+import { useDragDropManager } from "react-dnd";
+import throttle from "lodash/throttle";
+
+/**
+ * This is custom dom goodstrap event, it is used by Overlay to handle CloseOnParentScroll
+ */
+const fireGoodstrapScrollEvent = (
+    node: HTMLElement,
+    windowInstance = { dispatchEvent: (_event: Event) => true },
+) => {
+    windowInstance.dispatchEvent(
+        new CustomEvent("goodstrap.scrolled", {
```

Comment:
is `CustomEvent` supported by the Internet Explorer? (if not, we should add also polyfill).
Technically, we still should support IE11 (until v9 release).

Reply:
Yes in IE11 is supproted but needs to be constructed another ways https://stackoverflow.com/a/37566606

Reply:
custom-event-polyfill added to SDK-UI-Kit

Reply:
"custom-event" package added